extensions/eclipse-che-theia-plugin-ext/src/node/che-api-service.ts

```diff
@@ -17,14 +17,38 @@ export class CheApiServiceImpl implements CheApiService {
 
     private workspaceRestAPI: IRemoteAPI | undefined;
 
+    async getCurrentWorkspaceId(): Promise<string> {
+        return this.getWorkspaceIdFromEnv();
+    }
+
+    async getUserPreferences(filter?: string): Promise<Preferences> {
+        const apiClient = await this.getCheApiClient();
+        return await apiClient.getUserPreferences(filter);
+    }
+
+    async updateUserPreferences(update: Preferences): Promise<Preferences> {
+        const apiClient = await this.getCheApiClient();
+        return await apiClient.updateUserPreferences(update);
+    }
+
+    async replaceUserPreferences(preferences: Preferences): Promise<Preferences> {
+        const apiClient = await this.getCheApiClient();
+        return await apiClient.replaceUserPreferences(preferences);
+    }
+
+    async deleteUserPreferences(list?: string[]): Promise<void> {
+        const apiClient = await this.getCheApiClient();
+        return await apiClient.deleteUserPreferences(list);
```

Comment:
hello, why do we need this await ?

Reply:
https://eslint.org/docs/rules/no-return-await

Reply:
Good catch